.github/workflows/docker.yml

```diff
@@ -50,7 +50,7 @@ jobs:
           username: ${{ github.repository_owner }}
           password: ${{ secrets.GITHUB_TOKEN }}
       - name: Build and push
-        id: docker_build
+        id: docker_build1
```

Comment:
Can we use more descriptive ids to differentiate these? e.g. `docker_build_release` and `docker_build_debug`.

Reply:
Good point, done